modules/oe_theme_helper/oe_theme_helper.post_update.php

```diff
@@ -46,7 +46,7 @@ function oe_theme_helper_post_update_use_retina_image_styles(array &$sandbox): v
 /**
  * Change the region of the search block.
  */
-function oe_theme_helper_post_update_8201_change_region_of_search_block() {
+function oe_theme_helper_post_update_00001_change_region_of_search_block() {
```

Comment:
Let's update this to the new format with only the 00001.

Reply:
Same for the next 00002.

Reply:
done.